frontend/src/lib/components/Header/Header.svelte

```diff
@@ -0,0 +1,52 @@
+<script>
+	import All from '../Icons/All.svelte';
+	import Home from '../Icons/Home.svelte';
+	import Popular from '../Icons/Popular.svelte';
+	import Search from '../Icons/Search.svelte';
+	import HeaderAuth from './HeaderAuth.svelte';
+	import HeaderNoAuth from './HeaderNoAuth.svelte';
+
+	let auth = $state(false);
+</script>
+
+<div class="bg-neutral flex h-[75px] w-full flex-row items-center justify-between">
+	<!-- quibble icon -->
+	<div class="flex h-full items-center pl-5">
+		<h1 class="text-primary text-center text-3xl font-black">Quibble</h1>
+	</div>
```

Comment:
a change in design: use icon instead this (favicon one). add it to assets and use it.